### PR TITLE
chore: add a note about timescale.enable_chunk_skipping.

### PR DESCRIPTION
**Pull request by billy-the-fish**
This is for https://timescaledb.slack.com/archives/C4GT3N90X/p1730724406834719?thread_ts=1730713968.262729&cid=C4GT3N90X ( I can't see this param in Timescale Cloud so have only mentioned the self-hosted config.) and https://timescaledb.slack.com/archives/C5X6UVB24/p1732473441815019?thread_ts=1731620394.349939&cid=C5X6UVB24

Page structure is updated to match https://docs.timescale.com/api/latest/hypertable/create_hypertable/.

### CODE REVIEW

File: api/enable_chunk_skipping.md [hunk 1]
```diff
@@ -13,34 +13,36 @@ api:
...
-|`enabled`|BOOLEAN|Returns `true` when tracking is enabled, `if_not_exists` is `true`, and when a new entry is not
-added|
-
-<Highlight type="note">
- TimescaleDB supports min/max range tracking for the `smallint`, `int`,
- `bigint`, `serial`, `bigserial`, `date`, `timestamp`, and `timestamptz` data types.
-
-</Highlight>
+By default, this feature is disabled. To enable chunk skipping, set `timescale.enable_chunk_skipping = on` in
+`postgresql.cnf`. When you upgrade from a database instance that uses compression but does not support chunk
```
Comment (jonatas) [inline api/enable_chunk_skipping.md]:
```suggestion
`postgresql.conf`. When you upgrade from a database instance that uses compression but does not support chunk 
```

File: api/enable_chunk_skipping.md [hunk 2]
```diff
@@ -50,31 +52,28 @@ SELECT create_hypertable('conditions', 'time');
...
+|Column|Type|Description|
+|-|-|-|
+|`column_stats_id`|INTEGER|ID of the entry in the TimescaleDB internal catalog|
+|`enabled`|BOOLEAN|Returns `true` when tracking is enabled, `if_not_exists` is `true`, and when a new entry is not
+added|
 
- A [DROP COLUMN](https://www.postgresql.org/docs/current/sql-altertable.html#SQL-ALTERTABLE-DESC-DROP-COLUMN)
- on a column with statistics tracking enabled on it ends up removing all relevant entries
- from the catalog table.
+<Highlight type="note">
```
Comment (jonatas) [inline api/enable_chunk_skipping.md]: should we remove the empty highlight?

**Review by jonatas — APPROVED** [review]
Thanks for the improvement!

File: api/enable_chunk_skipping.md (hunk 2 above)
Comment (billy-the-fish) [reply api/enable_chunk_skipping.md]: Done

**Pull request merged**